app/javascript/styles/mastodon/admin.scss

```diff
@@ -778,7 +783,7 @@ a.name-tag,
 
 .speech-bubble {
   margin-bottom: 20px;
-  border-left: 4px solid $ui-highlight-color;
+  border-inset-inline-start: 4px solid $ui-highlight-color;
```

Comment:
Does this actually exist?

Reply:
https://github.com/mastodon/mastodon/blob/77bd32a26abbf8b49caf35dcdf170e312e48a277/stylelint.config.js#L16 can be used as a check for some of that stuff

Reply:
Oh, I messed up replacing absolute positioning properties (left/right) and failed to notice. My bad!